k8s/crds/kops.k8s.io_clusters.yaml

```diff
@@ -335,6 +335,28 @@ spec:
               cloudProvider:
                 description: The CloudProvider to use (aws or gce)
                 type: string
+              clusterAutoscaler:
+                description: ClusterAutoscaler defines the cluaster autoscaler configuration.
+                properties:
+                  balanceSimilarNodeGroups:
+                    description: 'BalanceSimilarNodeGroups makes cluster autoscaler treat similar node groups as one. Default: false'
+                    type: boolean
+                  enabled:
+                    description: 'Enabled enables the cluster autoscaler. Default: false'
+                    type: boolean
+                  expander:
+                    description: 'Expander determines the strategy for which instance group gets expanded. Supported values: least-waste, most-pods, random. Default: least-waste'
```

Comment:
Is there a reason for not supporting `priority` and `price` expanders? 

Obviously would require some more docs/links out to CA docs in both cases.

Reply:
Only because of simplicity. Shouldn't be a problem to add this later if anyone needs this. I'd rather not have that block this PR at least

Reply:
That seems reasonable enough.